django/librarian/utils/process_engine.py

```diff
@@ -270,9 +270,11 @@ def pdf_to_text_pdfium(content):
     pdf = pdfium.PdfDocument(content)
     for i, page in enumerate(pdf):
         text_page = page.get_textpage()
-        text += f"<page_{i+1}>\n"
-        text += text_page.get_text_range() + "\n"
-        text += f"</page_{i+1}>\n"
+        text_content = text_page.get_text_range()
```

Comment:
Could this affect pdf's where pages contain only OCR content? Like, if we completely omit pages without text but there's other content on there that can be OCR'd, will we still be able to get it?

Reply:
This is actually why I did this change. If there is no content in the whole document detected by this (non-OCR) method, it will fall back to using Azure OCR. Whereas if we added the empty `<page>` tags, the check on content length (to trigger forcing OCR) didn't work properly.